tests/x11/evolution/evolution_prepare_servers.pm

```diff
@@ -70,11 +74,14 @@ sub run() {
     send_key 'ret';
     save_screenshot;
 
+    systemctl 'status dovecot';
+    systemctl 'status postfix';
+
     select_console 'x11';
 }
 
 sub test_flags() {
-    return {milestone => 1};         # add milestone flag to save setup in lastgood VM snapshot
+    return {milestone => 1};    # add milestone flag to save setup in lastgood VM snapshot
```

Comment:
I remember there was a big PR which was removing such comments. It is generic openQA param which described at https://github.com/os-autoinst/openQA/blob/master/docs/WritingTests.asciidoc. No point to keep this explanation in every place it is used. Instead of proper alignment could you please just delete it ?

Reply:
I meant comment of course :) not test flag , it is up to you where to use a flag :)